_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1424,551 +1665,440 @@
+/area/ship/storage)
+"Hl" = (
+/obj/structure/sign/poster/contraband/random,
 /turf/closed/wall/mineral/titanium/nodiagonal,
 /area/ship/crew)
-"Kh" = (
-/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2{
-	dir = 1
-	},
-/obj/machinery/airalarm/directional/west,
-/obj/machinery/power/terminal,
-/obj/structure/cable,
-/obj/effect/turf_decal/industrial/warning,
-/turf/open/floor/plating,
-/area/ship/maintenance/port)
-"Ki" = (
-/obj/effect/landmark/start/paramedic,
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
-/obj/machinery/firealarm/directional/west,
-/obj/machinery/light_switch{
-	dir = 4;
-	pixel_x = -21;
-	pixel_y = -10
-	},
-/turf/open/floor/plasteel,
-/area/ship/crew)
-"Ko" = (
-/obj/machinery/light/directional/east,
+"HE" = (
+/obj/structure/rack,
 /obj/effect/turf_decal/industrial/outline/red,
-/obj/machinery/suit_storage_unit/inherit,
-/obj/item/clothing/suit/space/hardsuit/medical,
-/obj/item/clothing/mask/gas/sechailer,
-/obj/item/tank/internals/emergency_oxygen/engi,
+/obj/item/circuitboard/machine/ore_redemption,
+/obj/item/stack/sheet/glass/fifty,
+/obj/item/stack/sheet/metal/fifty,
+/obj/item/circuitboard/machine/rdserver,
+/obj/item/circuitboard/computer/rdconsole,
+/obj/item/radio/intercom/directional/west,
 /turf/open/floor/plasteel/tech,
 /area/ship/storage)
-"Kv" = (
-/obj/effect/turf_decal/trimline/opaque/red/filled/line{
-	dir = 4
-	},
-/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4{
+"HI" = (
+/obj/structure/table/reinforced,
+/obj/machinery/door/window/southleft{
 	dir = 8
 	},
-/turf/open/floor/plasteel/white,
-/area/ship/hallway/fore)
-"KD" = (
-/obj/structure/window/reinforced{
+/obj/machinery/button/door{
+	dir = 4;
+	id = "lobbydoors";
+	name = "Lobby Door Control";
+	normaldoorcontrol = 1;
+	pixel_x = 11;
+	pixel_y = -4
+	},
+/obj/machinery/door/firedoor/border_only{
 	dir = 8
 	},
-/obj/machinery/door/window/northright,
-/obj/machinery/power/terminal,
-/obj/structure/cable,
-/obj/machinery/light/small/directional/east,
-/turf/open/floor/plating,
-/area/ship/maintenance/port)
-"KR" = (
-/obj/structure/sink{
-	dir = 8;
-	pixel_x = 12
+/obj/effect/turf_decal/industrial/hatch/red,
+/obj/machinery/button/shieldwallgen{
+	dir = 4;
+	id = "traumashield";
+	name = "Lobby Holoshield";
+	pixel_x = -6;
+	pixel_y = 8
 	},
-/obj/effect/turf_decal/corner/opaque/white/mono,
-/obj/item/storage/box/pillbottles,
-/obj/item/storage/box/pillbottles,
-/obj/item/storage/box/medigels,
-/obj/item/storage/box/medigels,
-/obj/item/reagent_containers/dropper,
-/obj/item/reagent_containers/dropper,
-/obj/item/clothing/suit/toggle/labcoat/chemist/side,
-/obj/item/clothing/glasses/science,
-/obj/item/reagent_containers/glass/beaker/large,
-/obj/item/reagent_containers/glass/beaker/large,
-/obj/structure/closet/wall/white/chem{
-	dir = 1;
-	name = "Chemistry Locker";
-	pixel_y = -28
+/obj/machinery/button/door{
```

Comment:
Adjust the pixel offsets on these buttons a bit so they aren't hanging off the edge of the counter.